pkg/deployer/wait/daemonset.go

```diff
@@ -27,10 +27,10 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-func ForDaemonSetReadyByKey(cli client.Client, logger logr.Logger, key ObjectKey, pollInterval, pollTimeout time.Duration) (*appsv1.DaemonSet, error) {
+func ForDaemonSetReadyByKey(cli client.Client, ctx context.Context, logger logr.Logger, key ObjectKey, pollInterval, pollTimeout time.Duration) (*appsv1.DaemonSet, error) {
```

Comment:
I have noticed in Kubernetes codebase that `ctx context.Context` is usually the first parameter in a function definition. I guess it is a convention but it doesn't hurt to follow it :) Do you mind updating here and in a few other placed below so that `ctx context.Context` is passed as the first parameter?

Reply:
sure thing, I don't have a preference here so I updated the PR.